package.json

```diff
@@ -121,14 +121,29 @@
     "@storybook/addons": "^5.3.17",
     "@storybook/react": "^5.3.18",
     "@testing-library/react-hooks": "^3.2.1",
+    "@types/babel__core": "^7.1.7",
```

Comment:
Why add so many typing packages that we may not import them in the actual TypeScript code.

Reply:
OK